src/index.css

```diff
@@ -64,10 +67,10 @@ html {
 body {
 	background-color: var(--color-bg);
 	color: var(--color-text);
-	font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui,
-		helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;
-	/* font-size: 1.6rem;
-	line-height: 1.4; */
+	font-family: 'Edu VIC WA NT Beginner', cursive, -apple-system,
```

Comment:
After our accessibility workshop, I have been viewing web development with a different lens and I don't believe this font choice is accessible. On top of that, if the user's browser does not support that font and it defaults to cursive (the following backup font) it will become 100% inaccessible. [Choosing the right font](
https://pixelplex.io/blog/choosing-the-right-font-and-its-presentation/)

As a personal preference, I really like the handwriting style font and it goes with the theme of our app, but " Cursive or handwriting style fonts also make the text inaccessible, and hence they are not the best fonts to use for a website."

Reply:
I agree with Yire's assessment on the typography.

Reply:
Noted.  I'll look for something that more accessible by all users and implement the other changes recommended.

Reply:
Noted.  I'll look for something that more accessible by all users and implement the other changes recommended.  Also it looks like there's some confusion.  This is for chippy's separate component.  This comment should have been in the font styling instead.  I'll make the changes in the font styling pull request instead.